### PR TITLE
tikv: add storage flow control doc

### PR DESCRIPTION
**Pull request by Connor1996**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add the document about storage flow control 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/tikv/tikv/pull/10249

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by qiancai** [comment]
@Connor1996 Please add a tech reviewer for this PR. Thanks!

**Comment by TomShawn** [comment]
@tabokie @yiwu-arbug PTAL, thanks!

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -312,6 +312,35 @@ RocksDB 多个 CF 之间共享 block cache 的配置选项。当开启时，为
 + 默认值：系统总内存大小的 45%
 + 单位：KB|MB|GB
 
+## storage.flow-control
+
+在 storage scheduler 层进行流控代替 RocksDB 的 write stall 机制，可以避免 write stall 机制在大写入下卡住 raftstore/apply worker 导致 QPS drop 的问题。
```
Comment (tabokie) [inline tikv-configuration-file.md:317]:
```suggestion
在 scheduler 层进行流控代替 RocksDB 的 write stall 机制，可以避免 write stall 机制在大写入下卡住 raftstore/apply 线程导致 QPS drop 的问题。
```

**Review by tabokie — APPROVED** [review]

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -312,6 +312,35 @@ RocksDB 多个 CF 之间共享 block cache 的配置选项。当开启时，为
...
+
+### `enable`
+
++ 是否开启 flow control 流控机制。当开启时，会自动关闭 KvDB 的 write stall 和 RaftDB 的 write stall（除了 memtable）
++ 默认值：true
+
+### `memtables-threshold`
+
++ 当 KvDB 的 memtable 达到该阈值时，流控机制开始工作。
```
Comment (qiancai) [inline tikv-configuration-file.md:326]:
```suggestion
+ 当 KvDB 的 memtable 的个数达到该阈值时，流控机制开始工作。
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -312,6 +312,35 @@ RocksDB 多个 CF 之间共享 block cache 的配置选项。当开启时，为
 + 默认值：系统总内存大小的 45%
 + 单位：KB|MB|GB
 
+## storage.flow-control
+
+在 storage scheduler 层进行流控代替 RocksDB 的 write stall 机制，可以避免 write stall 机制在大写入下卡住 raftstore/apply worker 导致 QPS drop 的问题。
+
+### `enable`
+
++ 是否开启 flow control 流控机制。当开启时，会自动关闭 KvDB 的 write stall 和 RaftDB 的 write stall（除了 memtable）
```
Comment (qiancai) [inline tikv-configuration-file.md:321]: “除了 memtable” 这里可以再详细说明下吗？不会关闭 memtable 的 write stall?

**Review by qiancai — COMMENTED** [review]
@Connor1996 PTAL

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -312,6 +312,35 @@ RocksDB 多个 CF 之间共享 block cache 的配置选项。当开启时，为
 + 默认值：系统总内存大小的 45%
 + 单位：KB|MB|GB
 
+## storage.flow-control
+
+在 scheduler 层进行流控代替 RocksDB 的 write stall 机制，可以避免 write stall 机制在大写入下卡住 raftstore/apply 线程导致 QPS drop 的问题。
```
Comment (TomShawn) [inline tikv-configuration-file.md:317]:
```suggestion
在 scheduler 层进行流量控制代替 RocksDB 的 write stall 机制，可以避免 write stall 机制在写入量较大时卡住 Raftstore 或 Apply 线程导致 QPS 下降的问题。本节介绍 TiKV 流量控制机制相关的配置项。
```

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -312,6 +312,35 @@ RocksDB 多个 CF 之间共享 block cache 的配置选项。当开启时，为
...
+
+### `enable`
+
++ 是否开启 flow control 流控机制。当开启时，会自动关闭 KvDB 的 write stall 和 RaftDB 的除了 memtable 以外的 write stall
```
Comment (TomShawn) [inline tikv-configuration-file.md:321]: 如果 flow control 属于自然语言与"流量控制"意义无差别，建议在中文文档中尽量使用中文表达，只有尚无中文表达的专有名词可保留英文。
```suggestion
+ 是否开启 flow control 流控机制。当开启时，会自动关闭 KvDB 的 write stall 和 RaftDB 的除了 memtable 以外的 write stall
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**